### PR TITLE
Bugfix/#1032 ensure fspiop headers are sent

### PR DESCRIPTION
**Pull request by oderayi**
- Added FSPIOP headers to callbacks
- Added `docker-compose` artifacts for easier start-up during development
- ~Added unit tests for notification handler (WIP)~ Tests removed pending critical bug fixes (https://github.com/mojaloop/project/issues/1032).

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    networks:
+      - ba-mojaloop-net
+    depends_on:
+      - central-ledger
+      - kafka
+    healthcheck:
+      test: ["CMD", "sh", "-c" ,"apk --no-cache add curl", ";", "curl", "http://localhost:3000/health"]
+      timeout: 20s
+      retries: 10
+      interval: 30s
```
Comment (rmothilal) [inline docker-compose.yml]: Does bulk-api-adapter require ml-api-adapter?

File: docker/central-ledger/default.json [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+    "CREATE_RETRY_INTERVAL_MILLIS": 200,
+    "DEBUG": false
+  },
+  "MIGRATIONS": {
+    "DISABLED": false,
+    "RUN_DATA_MIGRATIONS": true
+  },
+  "AMOUNT": {
+    "PRECISION": 10,
+    "SCALE": 2
```
Comment (rmothilal) [inline docker/central-ledger/default.json]: this should be 
`
"PRECISION": 18,
"SCALE": 4
`

File: docker/bulk-api-adapter/default.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  },
+  "ENDPOINT_SECURITY":{
+    "TLS": {
+      "rejectUnauthorized": true
+    }
+  },
+  "MAX_FULFIL_TIMEOUT_DURATION_SECONDS": 300,
+  "AMOUNT": {
+    "PRECISION": 10,
+    "SCALE": 2
```
Comment (rmothilal) [inline docker/bulk-api-adapter/default.json]: this should be 
`
"PRECISION": 18,
"SCALE": 4
`

File: docker/ml-api-adapter/default.json [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+            },
+            "topicConf": {
+              "request.required.acks": "all"
+            }
+          }
+        }
+      }
+    }
+  }
+}
```
Comment (rmothilal) [inline docker/ml-api-adapter/default.json]: does bulk-api-adapter have a dependency on ml-api-adapter?

File: docker/wait-for/wait-for-ml-api-adapter.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "** STARTUP - Checking for ML-API-Adapter..."
+
+source /opt/wait-for/wait-for.env
+
+sh /opt/wait-for/wait-for-kafka.sh
+
+echo "** STARTUP - ML-API-Adapter successful!"
```
Comment (rmothilal) [inline docker/wait-for/wait-for-ml-api-adapter.sh]: does bulk-api have a dependency on ml-api?

File: docker-compose.yml (hunk 1 above)
Comment (oderayi) [reply docker-compose.yml]: Thanks @rmothilal. No it doesn't.

**Pull request merged**